File: PHP/ProjectEconomy/src/persistence/PessoaDao.php
<?php
	class PessoaDao{
		private $con;
		public function __construct(){
			$this->con = new Conexao();
			$this->con = $this->con->getConnection();
		}
		public function insert($pessoa){
			$sql = "insert into pessoa(nome,sobrenome,email,telefone) values(:1,:2,:3,:4)";
			$stmt = $this->con->prepare($sql);
			$stmt->bindValue(":1",$pessoa->get("nome"));
			$stmt->bindValue(":2",$pessoa->get("sobrenome"));
			$stmt->bindValue(":3",$pessoa->get("email"));
			$stmt->bindValue(":4",$pessoa->get("telefone"));
			$stmt->execute();
		}
		public function select(){
			$sql = "select * from pessoa";
			$stmt = $this->con->prepare($sql);
			$stmt->execute();
			return $stmt;
		}
	}
?>

File: JAVA EE/ProjectEconomy/src/persistence/PessoaDao.java
package persistence;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.util.ArrayList;
import java.util.List;

import connection.Conexao;
import entity.Pessoa;

public class PessoaDao {
	private Connection con;
	
	public PessoaDao() throws Exception{
		con = new Conexao().getConnection();
	}
	
	public void insert(Pessoa p) throws Exception{
		String sql = "insert into pessoa(nome,sobrenome,email,telefone) values (?,?,?,?)";
		PreparedStatement stmt = con.prepareStatement(sql);
		stmt.setString(1, p.getNome());
		stmt.setString(2, p.getSobrenome());
		stmt.setString(3, p.getEmail());
		stmt.setString(4, p.getTelefone());
		stmt.execute();
		stmt.close();
	}
	public List<Pessoa> select() throws Exception{
		String sql = "select * from pessoa";
		PreparedStatement stmt = con.prepareStatement(sql);
		ResultSet rs = stmt.executeQuery();
		Pessoa p;
		List<Pessoa> lista = new ArrayList();
		while(rs.next()) {
			p = new Pessoa();
			p.setId(rs.getInt("id"));
			p.setNome(rs.getString("nome"));
			p.setSobrenome(rs.getString("sobrenome"));
			p.setEmail(rs.getString("email"));
			p.setTelefone(rs.getString("telefone"));
			lista.add(p);
		}
		return lista;
	}
	public void delete(Pessoa p) throws Exception{
		String sql = "delete  from pessoa where id=?";
		PreparedStatement stmt = con.prepareStatement(sql);
		stmt.setInt(1, p.getId());
		stmt.execute();
		stmt.close();
	}
}
